viewer/vue-client/src/components/inspector/item-entity.vue

```diff
@@ -99,6 +102,9 @@ export default {
   },
   mounted() {
     this.$nextTick(() => {
+      if (this.isMaybeMagicShelfMark) {
+        this.$root.$emit('maybe-magic-shelf-mark-created', { path: this.actualParentPath, id: this.item['@id'] });
```

Comment:
Any specific reason for going via $root instead of using the store?

Reply:
This is just the event that signals to the inspector to maybe update the store.
I maybe should break out all the shelfControlNumber methods from inspector so that item-entity can do this by itself.

Reply:
Refactored